src/zone.ts

```diff
@@ -966,9 +959,11 @@ class Zone extends ServiceObject {
  *   const apiResponse = data[1];
  * });
  */
-import(localPath: string, callback: CreateChangeCallback) {fs.readFile(localPath, 'utf-8', (err, file) => {
+import(localPath: string): Promise<CreateChangeResponse>;
+import(localPath: string, callback: CreateChangeCallback): void;
+import(localPath: string, callback?: CreateChangeCallback): void|Promise<CreateChangeResponse> {fs.readFile(localPath, 'utf-8', (err, file) => {
```

Comment:
The formatting here seems weird 🤷‍♂️

Reply:
Indeed it does. Filed previously as https://github.com/google/ts-style/issues/214